string PartNumber
uint8 ID
---
string connectionstring
